src/components/organisms/Header/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { InnerWrapper, Wrapper } from '@/components/atoms';
+import styles from './header.module.scss';
+import { TodoList } from '@/pages';
+import { AddTodoInput } from '@/components/molecules';
+
+const Headerv2 = () => (
```

Comment:
Usually a Header it is UI element with navigation. This component is something like Form

Reply:
it is fixed;

"header" is now simpler and cleaner